actions/sequester/Quest2GitHub/Models/QuestWorkItem.cs

```diff
@@ -273,20 +274,25 @@ public static string BuildDescriptionFromIssue(GithubIssue issue, string? reques
             var newItem = QuestWorkItem.WorkItemFromJson(jsonDocument);
             linkedGitHubRepo = true;
             return newItem;
-        } catch (InvalidOperationException ex)
+        }
+        catch (InvalidOperationException ex)
         {
-            Console.WriteLine("Can't add closing PR. The GitHub repo is likely not configured as linked in Quest.");
+            Console.WriteLine($"""
```

Comment:
What does the """ do?

Reply:
This is the raw string literals: https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/strings/#raw-string-literals